forge-gui/res/cardsfolder/upcoming/krenkos_buzzcrusher.txt

```diff
@@ -0,0 +1,17 @@
+Name:Krenko's Buzzcrusher
+ManaCost:2 R R
+Types:Artifact Creature Insect Thopter
+PT:4/4
+K:Flying
+K:Trample
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ DBForEach | TriggerDescription$ When CARDNAME enters the battlefield, for each player, destroy up to one nonbasic land that player controls. For each land destroyed this way, its controller may search their library for a basic land card, put it onto the battlefield tapped, then shuffle.
+SVar:DBForEach:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBChoose | SubAbility$ DBDestroyAll
+SVar:DBChoose:DB$ ChooseCard | Defined$ You | ChoiceZone$ Battlefield | Amount$ 1 | Choices$ Land.nonBasic+RememberedPlayerCtrl | Optional$ True | RememberChosen$ True | ChoiceTitle$ Choose up to one nonbasic land this player controls.
```

Comment:
work with Imprint here
then there should be no need for both other RepeatEach since you can just have all RememberPlayer fetch together

Reply:
Did so to the best of my ability. Works as expected :)